src/main/java/uk/gov/gchq/palisade/exception/Error.java

```diff
@@ -145,29 +144,7 @@ public String toString() {
                 .toString();
     }
 
-    public RuntimeException createException() {
-        if (null == exceptionClass) {
-            return new PalisadeWrappedErrorRuntimeException(this);
-        }
-
-        if (null != simpleMessage) {
-            try {
-                final Constructor<? extends RuntimeException> constructor = exceptionClass.getConstructor(String.class);
-                return constructor.newInstance(simpleMessage);
-            } catch (final Exception e) {
-                LOGGER.error("Unable to recreate exception with message for error {}", this, e);
-            }
-        }
 
-        try {
-            return exceptionClass.getDeclaredConstructor().newInstance();
-        } catch (final Exception e) {
-            // ignore
-        }
-
-        return new PalisadeWrappedErrorRuntimeException(this);
-
-    }
 
 
     public static final class ErrorBuilder {
```

Comment:
Is this Error and ErrorFactory still used within the project?

Reply:
Sorry, I did not make this point very clear.  I'll try again.  We have two classes in the palisade-services with the name ErrorDetails.  There is an Error and ErrorFactory class.  These classes have overlapping functionality and I would suggest that they be considered for a separate refactoring exercise.  Depending on how this is refactored the classes Error and ErrorFactory in the common-repo may (or may not) end-up on the scrap heap.  Does that make more sense?